frontend/app/route-helpers/renew-ita-route-helpers.server.ts

```diff
@@ -96,6 +96,10 @@ export function validateRenewItaStateForReview({ params, state }: ValidateRenewI
     throw redirect(getPathById('public/renew/$id/ita/marital-status', params));
   }
 
+  if (contactInformation === undefined) {
+    throw redirect(getPathById('public/renew/$id/ita/contact-information', params));
+  }
```

Comment:
Since we're introducing the communication page in this PR

```suggestion
  if (communicationPreference === undefined) {
    throw redirect(getPathById('public/renew/$id/ita/communication-preference', params));
  }
```